binding.gyp

```diff
@@ -0,0 +1,25 @@
+{
+  "targets": [
+    {
+      "target_name": "elevator",
+      "include_dirs" : [
+        "src",
+        "<!(node -e \"require('nan')\")"
+      ],
+      'conditions': [
+
+        [ 'OS=="win"', {
+          "sources": [
+            "src/utils/v8utils.cpp",
+            "src/os/elevate.cpp",
```

Comment:
From looking through the code further down, it looks like this is all Windows-specific? Should "src/os/elevate.cpp" be at "src/win32/elevate.cpp" instead?